kubetest/util.go

```diff
@@ -88,8 +88,22 @@ func getLatestClusterUpTime(gcloudJSON string) (time.Time, error) {
 // getLatestGKEVersion will return newest validMasterVersions.
 // Pass in releasePrefix to get latest valid version of a specific release.
 // Empty releasePrefix means use latest across all available releases.
-func getLatestGKEVersion(project, zone, releasePrefix string) (string, error) {
-	res, err := control.Output(exec.Command("gcloud", "container", "get-server-config", fmt.Sprintf("--project=%v", project), fmt.Sprintf("--zone=%v", zone), "--format=value(validMasterVersions)"))
+func getLatestGKEVersion(project, zone, region, releasePrefix string) (string, error) {
+	cmd := []string{
+		"container",
+		"get-server-config",
+		fmt.Sprintf("--project=%v", project),
+		"--format=value(validMasterVersions)",
+	}
+
+	// zone can be empty for regional cluster
+	if zone != "" {
+		cmd = append(cmd, fmt.Sprintf("--zone=%v", zone))
+	} else if region != "" {
+		cmd = append(cmd, fmt.Sprintf("--region=%v", region))
+	}
+
+	res, err := control.Output(exec.Command("gcloud", cmd...))
```

Comment:
This still won't work without "beta" in case of region.
You need to also propagate the gkeCommandGroup:
https://github.com/kubernetes/test-infra/blob/master/kubetest/gke.go#L150

Reply:
updated again, not sure what's the most clean way to propagate the `gkeCommandGroup` flag all the way into extract function... as that's a global I'll just reference it here...